matrixbot/ldap.py

```diff
@@ -6,7 +6,7 @@
 
 
 
-import ldap as LDAP
+import ldap3 as LDAP
```

Comment:
Are you sure ldap3 is a drop-in replacement of ldap? 


Please update the requirements.txt with the corresponding package: `ldap3`

Reply:
I double-checked this change, and you're right it's not necessary. To test it I run the unit-tests and launched an instance of matrix-bot.

In addition to that I update a few other python files that were missing the Python3 shebang.